ui/package.json

```diff
@@ -9,7 +9,7 @@
     "@patternfly/react-core": "2.3.6",
     "@patternfly/react-table": "1.0.7",
     "dotenv": "^6.2.0",
-    "lodash": "4.17.11",
+    "lodash": "^4.17.15",
```

Comment:
```suggestion
"lodash": "4.17.15",
```

Reply:
It will not work and will cause the issue.